composer.json

```diff
@@ -11,8 +11,14 @@
             "email": "andrei@baibaratsky.com"
         }
     ],
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
```

Comment:
I wasn't able to do a `composer install` without it, I kept getting a dependency error, but it may not be necessary